docker/web-entrypoint.sh

```
@@ -62,12 +62,15 @@ rm -f "$APP_PATH/tmp/pids/server.pid"
 
 # Sync static assets from image to volume
 # This ensures new and updated files are copied to the persistent volume
-if [ -d "/tmp/public_assets" ]; then
+ASSETS_DIST="$APP_PATH/public_dist"
+if [ -d "$ASSETS_DIST" ]; then
   echo "📦 Syncing static assets to public volume..."
   # Remove old compiled assets to prevent stale files from persisting
   rm -rf $APP_PATH/public/assets
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)